recipes/iqtree/meta.yaml

```diff
@@ -6,11 +6,14 @@ package:
   version: {{ version }}
 
 build:
-  number: 0
+  number: 1
+  skip: true # [linux]
```

Comment:
TODO: revert this platform pin (currently on to prevent spending time testing on linux, since this PR is attempting to address build issues specific to darwin).